customer-commons/src/main/java/no/unit/nva/customer/model/CustomerMapper.java

```diff
@@ -1,28 +1,23 @@
 package no.unit.nva.customer.model;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
 import java.net.URI;
-import java.util.List;
+import java.util.Objects;
 import java.util.Optional;
 import java.util.UUID;
-import java.util.stream.Collectors;
 import no.unit.nva.customer.model.CustomerDto.Builder;
 import nva.commons.core.Environment;
-import nva.commons.core.JsonUtils;
 
-public class CustomerMapper {
+public final class CustomerMapper {
```

Comment:
This class has the functionality of just adding context to the DTOs. It will be renamed in later PR